addons/hr_holidays/models/hr_leave.py

```diff
@@ -461,6 +475,12 @@ def _compute_from_employee_id(self):
                 if holiday.employee_id and not holiday.holiday_status_id.with_context(employee_id=holiday.employee_id.id).has_valid_allocation:
                     holiday.holiday_status_id = False
 
+    @api.depends_context('uid')
+    def _compute_is_user_no_rights_manager(self):
+        user = self.env.user
+        self.is_user_no_rights_manager = user.has_group('hr_holidays.group_hr_holidays_responsible')\
```

Comment:
`is_user_only_responsible`? Just a suggestion, because is_user_no_rights_manager is a bit vague name for me